nf_core/pipelines/create/pipelinetype.py

```diff
@@ -11,7 +11,7 @@
 ## Choose _"nf-core"_ if:
 
 * You want your pipeline to be part of the nf-core community
-* You think that there's an outside chance that it ever _could_ be part of nf-core
+* Your pipeline has been accepted via the [nf-core proposal process](https://github.com/nf-core/proposals)
```

Comment:
Do you think that this GitHub link is self-explanitory enough? How about linking to https://nf-co.re/docs/tutorials/adding_a_pipeline/overview ?

Side note: looking at those docs, I think that we need a new page in that tutorial that is specifically about the proposal process, that goes in-between _Overview_ and _Creating a pipeline_. The existing sections are quite easy to miss.